packages/core/admin/admin/src/content-manager/components/Inputs/index.js

```diff
@@ -224,6 +224,8 @@ function Inputs({
       attribute={fieldSchema}
       autoComplete="new-password"
       intlLabel={{ id: label, defaultMessage: label }}
+      // in case the default value of the boolean is null, attribute.default doesn't exist
+      isNullable={inputType === 'bool' && fieldSchema.attribute?.default === undefined}
```

Comment:
What if the default value of the boolean field is null? Shouldn't we also allow this input to be set to `null` ?

WDYT about the following statement instead: `inputType === 'bool' && !fieldSchema.required`  or even this one `inputType === 'bool' && (fieldSchema.default === undefined || fieldSchema.default === null)`

Reply:
I think these are 2 good remarks:

- `inputType === 'bool' && !fieldSchema.required`: Imo this case should be handled earlier already (when creating the field). It shouldn't be possible to create a required field with a `null` default value. Also, it might make sense for entities in a draft state, to be nullable even if they are required, eg. if one editor wants to signal he/ she is not sure about a certain value.

- `inputType === 'bool' && (fieldSchema.default === undefined || fieldSchema.default === null)` The latter case doesn't exist yet. Do you think it would make sense to add it in case at some point we start passing down `default=null`?